tools/podmanpreview.sh

```diff
@@ -1,7 +1,23 @@
 #!/usr/bin/env sh
+#
+# Copyright (c) 2021 Red Hat, Inc.
+# This program and the accompanying materials are made
+# available under the terms of the Eclipse Public License 2.0
+# which is available at https://www.eclipse.org/legal/epl-2.0/
+#
+# SPDX-License-Identifier: EPL-2.0
+#
+
 set -ex
 
-podman run --rm -ti \
+if command -v podman
```

Comment:
it looks like all this commands have almost 95% of copy/paste

would it be possible to include most of the code in a shared file ?

or what about utility where you provide the command you want
./che-docs <linkchecker|preview|>


or even more generic way of running it:
${RUNNER} exec -ti che-docs "./tools/<command>.sh"

where command is like $1 parameter, etc